src/api/route-services/pipeline-response.js

```diff
@@ -4,16 +4,17 @@ const logger = require('../../utils/logging');
 
 const ExperimentService = require('./experiment');
 const PlotsTablesService = require('./plots-tables');
+
 const pipelineStatus = require('../general-services/pipeline-status');
 
-const experimentService = new ExperimentService();
 const plotsTableService = new PlotsTablesService();
+const experimentService = new ExperimentService();
 
 const pipelineResponse = async (io, message) => {
   await validateRequest(message, 'PipelineResponse.v1.yaml');
 
   // Fail hard if there was an error.
-  const { response: { error }, input: { experimentId, taskName } } = message;
+  const { response: { error }, input: { experimentId, taskName, sampleUuid } } = message;
```

Comment:
`sampleUuid` is added as a task input so we can associate a result with a particular sample uuid (previously the result had no reference to the sample it was run on)